Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+# MRTK Scene Understanding Demo
+
+This is an __experminental__ prelease that integrates [Scene Understanding](https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding) (SU) as a MRTK.
+
+SU - in a nutshell, will return a semantic representation of scene entities as well as their geometric forms on __HoloLens 2__.
+
+## Goals and features of the demo
+* Demo scene with visualization of Scene Objects with options for configuring the observer
+* Example script that excersises the event system demonstrating common tasks
+* Save device room data for offline development
+* In-editor workflow for developing with previously saved room data (.bytes files)
+* For the first time, access to the highest possible resolution of the world mesh!
+
+## Observer overview
+
+When asked, the SU observer will return Scene Objects with attributes useful for mixed world development. What is returned is dependent on the configuration of the observer when fetched. For instance, if you want the occlusion mask, the observer must be configured to generate quads.
+
+It is expected the majority of "real" applications will never show the underlying data in raw form.
+
+## Background and development notes
+
+These links cover the underlying technology the Spatial Awareness service is using behind the scenes.
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding-sdk
+
+The Scene Understanding observer inherits from older spatial awareness concepts so the implementation is and odd fit in places.
+
+Eventually this system should replace the existing SpatialAwareness observer.
+
+Scene Understanding is a __platform specific__ technology. Spatial Awareness is meant to cover _platform agnostic_ capabilities. With few data points, it's likely we don't have the proper abstraction yet. 
+
+## Prerequisites
+
+Ensure build settings' platform is set to UWP.
+
+Ensure your project is configured to use MSBuild.
+
+You may need to restart unity if you get errors about missing assembly references. This should trigger download of packages.
+
+![Manage MRTK Configuration](Images/ConfigureMRTKMSBuildMenu.jpg)
+
+![MSBuild enabled](Images/MRTKProjectConfiguratorMSBuildEnabled.jpg)
+
+
+## Demonstration and debugging SU
+
```

Comment:
It would be helpful to add code snippets for someone getting started. The example scene is very useful but what are the simple lines of code to even get basic access to the observer/system. Maybe like Golden path/typical path scenario with code snippets. See following as example
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/SpatialAwareness/UsageGuide.html

Reply:
It would also be good to callout the key platform classes to utilize like IMixedRealitySpatialAwarenessSceneUnderstandingObserver  and add an xref so people are linked to the documentation page for the class/interface